cloudferrylib/os/actions/verify_vms.py

```diff
@@ -70,25 +70,20 @@ def run(self, **kwargs):
                 if src_inst_info['name'] != dst_cmp_inst['name']:
                     LOG.warning("Wrong name of instance %s on DST",
                                 src_inst_id)
-                    failed_vms.append(src_inst_id)
                 if (src_inst_info['flav_details'] !=
                         dst_cmp_inst['flav_details']):
                     LOG.warning("Wrong flav_details of instance %s on DST",
                                 src_inst_id)
-                    failed_vms.append(src_inst_id)
                 if src_inst_info['key_name'] != dst_cmp_inst['key_name']:
                     LOG.warning("Wrong key_name of instance %s on DST",
                                 src_inst_id)
-                    failed_vms.append(src_inst_id)
                 if (sorted(src_inst_info['interfaces']) !=
                         sorted(dst_cmp_inst['interfaces'])):
                     LOG.warning("Wrong interfaces of instance %s on DST",
                                 src_inst_id)
-                    failed_vms.append(src_inst_id)
```

Comment:
Interfaces must always stay the same, it's one of the most critical points to check and defines migration success more than all other attributes combined.